easybuild/easyconfigs/m/MAGeCK-VISPR/MAGeCK-VISPR-0.5.6-foss-2023a.eb

```diff
@@ -0,0 +1,153 @@
+# #
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Jordi Blasco <jordi.blasco@hpcnow.com>; Guillermo AR Piccoli <guillermo.piccoli@hpcnow.com>
+# License::     GPL-v3.0
+# #
+
+easyblock = 'PythonPackage'
+
+name = 'MAGeCK-VISPR'
+version = '0.5.6'
+
+homepage = 'https://bitbucket.org/liulab/mageck-vispr/'
+description = """MAGeCK-VISPR is a comprehensive quality control, analysis and visualization workflow for CRISPR/Cas9
+  screens The workflow combines the MAGeCK algorithm to identify essential genes from CRISPR/Cas9 screens considering
+  multiple conditions with VISPR to interactively explore results and quality control in a web-based frontend."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+source_urls = ['https://bitbucket.org/liulab/%(namelower)s/get']
+sources = ['v%(version)s.tar.gz']
+checksums = ['b6bdd59d34f9bbb548c88de8948e2aacf7e461a62a74c1b8ef8a73b9b64b2e92']
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('Meson', '1.2.3', '', ('GCCcore', '13.2.0'))
+]
+
+download_dep_fail = True
+use_pip = True
+sanity_pip_check = True
+
+options = {'modulename': 'mageck_vispr'}
+
+exts_defaultclass = 'PythonPackage'
+
+exts_default_options = {
+    'download_dep_fail': True,
+    'sanity_pip_check': True,
+    'source_urls': [PYPI_SOURCE],
+    'use_pip': True,
+}
+
+exts_list = [
```

Comment:
Almost all (maybe all?) of the extensions are part of Python bundles we have already. In particular Python-bundle-PyPI and SciPy-bundle. The hatchling + packaging stuff should be added as builddeps.